### PR TITLE
Fix #1360: FIDO2: Demo application structure

### LINKED ISSUE #1360: FIDO2: Demo application structure
Part of https://github.com/wultra/powerauth-server/issues/1315

### PR DESCRIPTION
**Pull request by jnpsk**
Publishing the fido2 demo app in smaller chunks. This is just the initial structure, mainly to validate the project setting

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -78,6 +78,7 @@
         <module>powerauth-fido2</module>
         <module>powerauth-java-server</module>
         <module>powerauth-admin</module>
+        <module>powerauth-fido2-demo</module>
```
Comment (banterCZ) [inline pom.xml:81]: Not crucial, but wondering whether we should not use alphabetical order or at least group fido functionality together.

File: powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.wultra.app.powerauthfido2demo;
```
Comment (banterCZ) [inline powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java:1]: Missing a copyright header.

**Review by banterCZ — APPROVED** [review]
Looks good.

File: pom.xml (hunk 1 above)
Comment (jnpsk) [reply pom.xml:79]: good idea

File: powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.app.powerauthfido2demo;
```
Comment (romanstrobl) [inline powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java:19]: The package name seems quite long, can we either simplify it or split into two subfolders?

**Review by romanstrobl — DISMISSED** [review]
A nice start.

File: powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java (hunk 3 above)
Comment (jnpsk) [reply powerauth-fido2-demo/src/main/java/com/wultra/app/powerauthfido2demo/PowerauthFido2DemoApplication.java]: set to `com.wultra.app.powerauth.fido2.demo`

**Comment by jnpsk** [comment]
Will be part of powerauth-tests repository

**Pull request closed without merging**